influxql/ast.go

```diff
@@ -137,6 +135,13 @@ type Statement interface {
 	RequiredPrivileges() ExecutionPrivileges
 }
 
+// HasDefaultDB provides an interface to get the default DB from a Statement.
+type HasDefaultDB interface {
```

Comment:
I prefer the shorter.  @benbjohnson @corylanou thoughts?  Also curious what everyone thinks about naming an interface `Has...`.